### ISSUE TITLE
Clicking on full screen throws exception if toolbar is customized

### ISSUE THREAD

**Issue by inssein** [froala/wysiwyg-editor#3483]
**Fiddle:** https://jsfiddle.net/5wvnqh62/2/

When you have an old style toolbar (without moreText), and you fullscreen, froala throws an exception.

**Comment by inssein** [comment]
Any update here? It seems like Froala went ahead and did a GA with known issues so far..

**Comment by jmiserez** [comment]
The issue is that Froala expects the moreText toolbar buttons to exist, and an exception occurs when they don't.

**Comment by jmiserez** [comment]
The fix is a check if $moreButton exists:
/**
     * Apply styles to button groups
     */
    function more($moreButton) {
      // Align the hidden toolbar with respect to its corresponding more button
      // Set its width to cover the complete editor width
      var $moreToolbar = editor.$tb.find(".fr-more-toolbar[data-name=\"".concat($moreButton.attr('data-group-name'), "\"]"));

**Comment by jmiserez** [comment]
A simple check if $moreButton is defined in more() fixes it:

```
/**
 * Apply styles to button groups
 */
function more($moreButton) {
  if (!$moreButton.length) {
    return;
  } 
  // rest of function
```

**Comment by casualuser** [comment]
fixed in 3.0.4
